go

```diff
@@ -65,7 +65,9 @@ BASE_PROOFER_OPTS = {
     %r{https://github.com/18F/Accessibility_Reviews}i,
     %r{https://github.com/18F/DevOps}i,
     %r{https://github.com/18F/handbook}i,
-    %r{https://github.com/18F/writing-lab}
+    %r{https://github.com/18F/writing-lab},
+    # https://github.com/gjtorikian/html-proofer/issues/118
+    '#'
```

Comment:
The "return to top" link.

https://travis-ci.org/18F/before-you-ship/builds/225712602#L236-L237